content/es/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,41 @@
+---
+title: Tutorial interactivo - Implementando una aplicación
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <br>
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                Para interactuar con la Terminal, use la versión de escritorio / tableta
+            </div>
+
+            <div class="katacoda__box" id="inline-terminal-1" data-katacoda-lang="es" data-katacoda-id="kubernetes-bootcamp/7" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-font="Roboto" data-katacoda-fontheader="Roboto Slab" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">Continuar con Modulo 3<span class="btn__next">›</span></a>
```

Comment:
```suggestion
                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">Continuar con el Módulo 3<span class="btn__next">›</span></a>
```

M**ó**dulo is accentuated in Spanish.